5.2/5.2.9/test/__init__.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+import importlib
+import os
+import tempfile
+from pathlib import Path
+import sys
+import unittest
+
+script_file_name = 'docker-initialize.py'
+class_name = 'Environment'
+input_file_names = dict(
+    zope_conf="zope.conf",
+    zeopack_conf="zeopack",
+    zeoserver_conf="zeo.conf"
+)
+output_file_names = dict(
+    custom_conf="custom.cfg",
+    cors_conf="999-additional-overrides.zcml",
+    zeopack_conf=input_file_names['zeopack_conf'],
+    zeoserver_conf=input_file_names['zeoserver_conf']
+)
+
+test_path = Path(__file__).parent
+expected_files_path = test_path.joinpath("expected-files")
+root_path = test_path.parent
+script_files = list(root_path.glob('**/' + script_file_name))
+
+if not expected_files_path.exists():
+    expected_files_path.mkdir()
+
+
+def _import_class(file):
```

Comment:
I would prefer to use `subprocess.run()` here but that would require changes to `docker-initialize.py` which I'd like to postpone to a separate pull request.